settings.gradle

```diff
@@ -26,4 +27,5 @@ include 'contentgrid-spring-boot-platform'
 include 'contentgrid-spring-data-rest'
 include 'contentgrid-spring-integration-events'
 include 'contentgrid-spring-querydsl'
-include 'contentgrid-spring-swagger-ui'
\ No newline at end of file
+include 'contentgrid-spring-swagger-ui'
+include 'contentgrid-spring-audit-logging'
```

Comment:
Technically, it's (going to be) broader than logging. Shouldn't we just call this module `contentgrid-spring-audit` ?

Reply:
I did not want this module to be confused with a future module that will handle the "audit" metadata (`(created|modified) (at|by)`), which is a concern that is completely separate from audit logging.

The "logging" does not refer to writing it to an slf4j logger only.